spk/maraschino-custom/src/maraschino-custom.sc

```diff
@@ -0,0 +1,6 @@
+[maraschino-custom]
+title="Maraschino-custom"
```

Comment:
To keep in line with the other Custom packages, change this to `Maraschino Custom`. Same with the `Desc` entry, on the next line.